internal/receiver/databricksreceiver/metadata.yaml

```
@@ -62,3 +94,1118 @@ metrics:
       value_type: int
     attributes:
       [job_id, task_id]
+
+  databricks.spark.codegenerator.compilationtime:
```

what about snake casing instead of concatenation?

```suggestion
  databricks.spark.code_generator.compilation_time:
```

---

Fixed (there were a lot -- previous naming came directly from Spark).